I'm getting ERROR IN CALL TO FUNCTION

Fatal Error: Error in call to extension function {public org.w3c.dom.NodeList com.occamlab.te.TECore.request(org.w3c.dom.Document,java.lang.String) throws java.lang.Throwable}: Exception in extension function java.net.UnknownHostException: C; SystemID: file:/C:/Users/vedat.yurt/teamengine/teamengine-web/target/te-base/work/src_main_scripts_ctl_main.xml/csw$Main.test; Line#: 40; Column#: -1
Eyl 09, 2015 4:35:47 PM com.occamlab.te.TECore executeTest
SEVERE: Error in call to extension function {public org.w3c.dom.NodeList com.occamlab.te.TECore.request(org.w3c.dom.Document,java.lang.String) throws java.lang.Throwable}: Exception in extension function java.net.UnknownHostException: C
Test csw:Main Failed
Report file "C:\Users\vedat.yurt\teamengine\teamengine-web\target\te-base\users\vedat.yurt\2702fdbb-291b-4713-a2f5-2c1c6a5994ec\report.html" created!
Tests run: 1, Failures: 1, Errors: 0, Skipped: 0, Time elapsed: 7.242 sec <<< FAILURE! - in com.occamlab.te.web.VerifyTestSuite
executeStartingTest(com.occamlab.te.web.VerifyTestSuite)  Time elapsed: 7.242 sec  <<< FAILURE!
java.lang.AssertionError: Unexpected result. expected:<5> but was:<6>
        at org.junit.Assert.fail(Assert.java:88)
        at org.junit.Assert.failNotEquals(Assert.java:834)
        at org.junit.Assert.assertEquals(Assert.java:645)
        at com.occamlab.te.web.VerifyTestSuite.executeStartingTest(VerifyTestSuite.java:144)


Results :

Failed tests:
  VerifyTestSuite.executeStartingTest:144 Unexpected result. expected:<5> but was:<6>

Tests run: 10, Failures: 1, Errors: 0, Skipped: 0

[INFO] ------------------------------------------------------------------------
[INFO] Reactor Summary:
[INFO]
[INFO] TEAM Engine ........................................ SUCCESS [  2.517 s]
[INFO] TEAM Engine - Tomcat Realm ......................... SUCCESS [  5.865 s]
[INFO] TEAM Engine - Shared Resources ..................... SUCCESS [  7.317 s]
[INFO] TEAM Engine - Core Module .......................... SUCCESS [ 26.962 s]
[INFO] TEAM Engine - Service Providers .................... SUCCESS [  8.307 s]
[INFO] TEAM Engine - Web Application ...................... FAILURE [ 30.387 s]
[INFO] TEAM Engine - Console Application .................. SKIPPED
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time: 01:21 min
[INFO] Finished at: 2015-09-09T16:35:47+03:00
[INFO] Final Memory: 39M/429M

--- a/teamengine-web/src/test/java/com/occamlab/te/web/VerifyTestSuite.java
+++ b/teamengine-web/src/test/java/com/occamlab/te/web/VerifyTestSuite.java
@@ -82,7 +82,7 @@ public class VerifyTestSuite {
     urlDir = new File(urlDir.toString().split("target")[0] + "src" + System.getProperty("file.separator")
             + "main" + System.getProperty("file.separator") + "scripts" + System.getProperty("file.separator")
             + "server" + System.getProperty("file.separator") + "csw.xml");
-    kvp = "capabilities.url=file://" + urlDir.toString();
+    kvp = "capabilities.url=file:" + System.getProperty("file.separator") + urlDir.toString();
   }
 
   @AfterClass

